cocos2d/CCDirector.h

```diff
@@ -66,13 +66,13 @@
 /** Stops the animation. Nothing will be drawn. The main loop won't be triggered anymore.
  If you want to pause your animation call [pause] instead.
  */
--(void) stopAnimation;
+-(void) stopRunLoop;
```

Comment:
The comment above will need to be changed to reflect the functionality and so the documentation doesn't conflate "animation" and "runloop"